### ISSUE TITLE
Issue in creating a periodogram

### ISSUE THREAD

**Issue by lopamudra-13** [lightkurve/lightkurve#1347]
#### Problem description
I get an error everytime I try to generate a periodogram from the lightcurve. 
#### Example
`
![pg_err](https://github.com/lightkurve/lightkurve/assets/138101744/818d9794-c483-4d75-a280-53576e705c4f)

`
```python
import lightkurve
search_result = lk.search_lightcurve('KIC5000307',quarter=(0,1,2,3,4,5,6,7,8,9,10,11,12))
lc = search_result.download_all().stitch()
lc = lc.normalize().remove_nans().remove_outliers() 

pg = lc.normalize().to_periodogram(normalization='psd')
```

#### Expected behavior
This issue started once I updated astropy. The same code was working just fine before that. How can I remove this error?

#### Environment

-  platform (e.g. Linux, OSX, Windows):
-  lightkurve version (e.g. 1.0b6):
-  installation method (e.g. pip, conda, source):

**Comment by rebekah9969** [comment]
Hello  @lopamudra-13 

Thank you for posting this issue. 

Would you mind telling me what version of lightkurve you are using please?

Thank you.

**Comment by keyuxing** [comment]
This issue is similar to #1344, and has already been fixed by #1342. However, it still exists since Lightkurve v2.4.1 is yet to be released, which contains this fix.

As a temporary workaround, you can run this command to install the dev version of Lightkurve v2.4.1:
```shell
pip install git+https://github.com/lightkurve/lightkurve.git
```
It should solve the problem you're experiencing.

**Comment by Nschanche** [comment]
I think this issue has been resolved with PR #1342 . I'm going to close this issue now, but if you have further problems, you can open it back up.
